docs/how-to/store-making-storage-deals.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Making Storage Deals
+description: Learn how to use Filecoin tokens to pay for storage.
+---
+
+# Making Storage Deals
+
+Once a client has prepared their data and added tokens to their wallet, they can make storage deals with miners. There are several distinct phases:
+
+### Discovering storage miners
+
+In the discovery phase, clients identify the miners who might be able to store their data. They start by querying the chain for the list of registered miners, and sends each miner a request for its storage ask (short for *asking price*).
+
+The storage ask contains details such as price, duration, and available storage capacity. Clients use these details to choose their preferred miner.
+
+*Note: In the near future, 3rd party reputation systems will allow clients to check for geography, miner reputation, and other factors.*
+
+### Negotiating the storage deal
+
+Next, the client locks up sufficient funds in their wallet to pay for the deal. The client also calculates the unique piece commitment (CommP) for the data it intends to store, which is used for confirmation in later phases.
```

Comment:
Would you like to say a little more here about what a CommP is?

Reply:
Added a bit here.